test/specs/views/Items/Item-test.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import {Item} from 'stardust';
+
+describe('Item', () => {
+  it('has children', () => {
+    let renderedItem = render(
+      <Item description='discription'>
```

Comment:
I've added an issue to extend conformance tests to handle this so we only have to write it once.  See #71.  We can safely remove this test for that issue.

For future reference, use a `find*()` method to assert the presence of something in the hierarchy.  Avoid writing tests that depend on component structure.  The find method will fail the test if it does not find exactly one occurrence.  So, no matches or more than one will fail.

```jsx
let child = faker.hacker.phrase();
render(<Item description='foo'>{child}</Item>).findText(child);
```

Tests that depend on the structure of the component are more apt to fail for the wrong reason.  In this case, if the children were rendered somewhere else, this test would fail even though the component is still rendering children.